runtime/syntax/vim.vim

```diff
@@ -166,7 +166,7 @@ syn match vimNumber	"-\d\+\%(\.\d\+\%([eE][+-]\=\d\+\)\=\)\="  skipwhite nextgro
 syn match vimNumber	"\<0[xX]\x\+"		       skipwhite nextgroup=vimGlobal,vimSubst,vimCommand,vimComment
 syn match vimNumber	"\%(^\|\A\)\zs#\x\{6}"             	       skipwhite nextgroup=vimGlobal,vimSubst,vimCommand,vimComment
 
-" All vimCommands are contained by vimIsCommands. {{{2
+" All vimCommands are contained by vimIsCommand. {{{2
```

Comment:
Changes to this file are supposed to be emailed to Dr. Chip.

Reply:
I assume that is @cecamp then?

I think it is better/good to have patches here, which allowed me to find e.g. https://github.com/vim/vim/issues/2450 in the first place.

Reply:
> I think it is better/good to have patches here

That is not The Vim Way.

Reply:
I've emailed Dr. Chip.